README.md

```diff
@@ -52,3 +52,66 @@ The script which is responsible for streaming the raw CAN data into the database
 Currently to use the script run main and when prompted input the path to the .mf4 file you want to have uploaded.<br>
 
 As we do not have means to collect context data yet, we use a dummy context with an id of 1. This is gross and bad. We will be removing that as soon as we have a way to query the user for context data.<br>
+
+## Web UI 
+
+This react-strap based website is responsible for gathering all the context relating to the data we gather.<br>
+
+Ideally, mechies will just download the files and run it on their computer. This includes running both the website and backend api that goes along with it.<br>
+
+In the future, I hope to host at least the backend restful api, and eventually the website itself, on a web server, but that is not going to happen for a long time. To anyone that works on this in the future, if this message is still here, that means I never went back and fixed the mess I made, and for that I sincerely apologize for. This was meant to be quick and dirty, but I attempted to follow best practices and self document the code where I could. <br>
+
+## RESTful API
+
+The RESTful API is built off python using Flask. <br>
+
+Someday, this will be hosted on a central server, but for now it runs locally. To run this application, see directions below under **Running the Server**. <br>
+
+For any future developers, standard practices should be followed when building this API. This includes separating calls into different classes and using the `GET`, `PUT`, `POST`, and `DELETE` calls correctly. Additionally, any changes to the url path for a call should be updated in the ServerPath.json file.
+
+### Download Instructions
+
+
+#### Node JS
+The first step is to install Node JS if you do not already have it installed.<br>
+
+1. Follow the download instructions found here [Node JS](https://nodejs.org/en/download/package-manager)
+2. Verify download by running `npm --version` or `node -v`<br>
+
+After Node JS has been installed, clone this repo onto your computer. If you have installed, open the project in VS Code. If you don't have in installed, just follow along in the command window. <br>
+
+Also run `npm install react` and `npm install reactstrap`
+
```

Comment:
Please add a note about getting/updating Python. Most people dont even know what a PATH variable is let alone how to set python in theirs.